apps/web/src/app/globals.css

```
@@ -3,7 +3,11 @@
 @source "../../../../packages/ui";
 
 @theme {
-  --font-pretendard: var(--font-pretendard), ui-sans-serif, system-ui, sans-serif;
+  --font-pretendard:
+    "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont,
```

Stylelint (17.7.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)